src/utils/DateUtils.js

```diff
@@ -131,22 +128,22 @@ module.exports = {
      * @param {Date}
      * @return {String}
      */
-    formatDate(date, format) {
+    formatDate(date, format, traslation) {
         let year = date.getFullYear()
         let month = date.getMonth() + 1
         let day = date.getDate()
-        let monthName = this.getMonthName(date.getMonth())
+        let monthName = this.getMonthName(date.getMonth(), traslation.months.original)
         let str = format
+            .replace(/S/, this.getNthSuffix(date.getDate()))
+            .replace(/D/, this.getDayNameAbbr(date, traslation.days))
```

Comment:
The suffix and day name rules were originally set to run last so as to not be overridden, for instance a suffix added to the third of the month would result in `3rd` and would then get the `d` replaced by the day rule.
Was there another reason you moved these to the top?